### PR TITLE
MAN-282 External link entity

### PR DESCRIPTION
**Pull request by nomadicoder**
Issue #MAN-282

- Add external_link entity
- Add external_link dashboard
- Make categorizable

### CODE REVIEW

File: app/dashboards/external_link_dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+class ExternalLinkDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    versions: Field::HasMany.with_options(class_name: "PaperTrail::Version"),
```
Comment (bibliotechy) [inline app/dashboards/external_link_dashboard.rb]: We don't seem to be doing this in any other dashbaord, so I think this is out of place.

File: app/dashboards/external_link_dashboard.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  COLLECTION_ATTRIBUTES = [
+    :id,
+    :title,
+    :link,
+  ].freeze
+
+  # SHOW_PAGE_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's show page.
+  SHOW_PAGE_ATTRIBUTES = [
+    :versions,
```
Comment (bibliotechy) [inline app/dashboards/external_link_dashboard.rb]: We don't seem to be doing this in any other dashbaord, so I think this is out of place.

File: app/dashboards/external_link_dashboard.rb (hunk 1 above)
Comment (nomadicoder) [reply app/dashboards/external_link_dashboard.rb]: - Removed.
- Also changed the class the dashboard inherits from.

File: app/dashboards/external_link_dashboard.rb (hunk 2 above)
Comment (nomadicoder) [reply app/dashboards/external_link_dashboard.rb]: - Removed.
 -  Also changed the class the dashboard inherits from.

File: app/dashboards/external_link_dashboard.rb (hunk 1 above)
Comment (bibliotechy) [reply app/dashboards/external_link_dashboard.rb]: I still see this line

File: app/dashboards/external_link_dashboard.rb (hunk 2 above)
Comment (bibliotechy) [reply app/dashboards/external_link_dashboard.rb]: @skng5 I still see this line.

File: app/dashboards/external_link_dashboard.rb (hunk 1 above)
Comment (bibliotechy) [reply app/dashboards/external_link_dashboard.rb]: @skng5 :point_up:

Comment (nomadicoder) [reply app/dashboards/external_link_dashboard.rb]: I see it too. I missed a step pushing it to GitHub. It should be there now.

File: app/dashboards/external_link_dashboard.rb (hunk 2 above)
Comment (bibliotechy) [reply app/dashboards/external_link_dashboard.rb]: @skng5 I still see this line

Comment (bibliotechy) [reply app/dashboards/external_link_dashboard.rb]: Which would now cause an error because the attribute is not defined for the dashboard.

**Pull request merged**